### ISSUE TITLE
Direction of arrows for undirected graphs -nx.draw(G, arrows=True)- misleading

### ISSUE THREAD

**Issue by dtekinoglu** [networkx/networkx#5466]
When I create an undirected graph and use `nx.draw()` function with argument `arrows=True`, arrows appear only in one direction even though I explicitly add edges (u,v) and (v,u) -which is actually not necessary in case of undirected graphs.

**Current Behavior**

`G=nx.Graph()`
`G.add_edge(2,0)`
`G.add_edge(0,1)`
`G.add_edge(1,0)`
`nx.draw(G, arrows=True, with_labels=True)`
![image](https://user-images.githubusercontent.com/79102788/161607567-828436e3-13fc-4329-be21-e49d3e47c2a0.png)

**Expected Behavior**

Arrows should appear in both directions for undirected graphs.

If this is not a feature but something to be improved, I can gladly work on this. @dschult @rossbar @MridulS

**Comment by dschult** [comment]
The default is to draw arrows with single arrowhead. 
But one can set the `arrowstyle` to "<->" if that is desired.
I guess you are saying that a more reasonable arrowstyle default 
for undirected graphs would be "<->".

**Comment by dtekinoglu** [comment]
Thank you @dschult , I am working on this then.

**Comment by jarrodmillman** [comment]
I am not sure it makes sense to draw arrows for an undirected graph.  Maybe it should at least raise a warning.  And if we do draw arrows, I agree that we should use "<->".

**Comment by 0ddoes** [comment]
According to Complex Network Theory, I don't think there should be double headed arrows for undirected graphs. If given <->, then there would not be any difference between ```nx.Graph()``` and ```nx.DiGraph()``` once arrows = ```True```. Or in other words, it might remove the difference between undirected and bidirectional graphs. Undirected is supposed to be showing connectedness while directed shows the mutual relations. And while studying actual datasets it could prove a problem. 

My suggestion is that @dtekinoglu could perhaps work on ```nx.draw()``` such that it recognises that it is undirected and gives a default ```None``` value instead. At present it follows 
```python
If `True`, draw arrowheads with FancyArrowPatches (bendable and stylish)
```
Something else I noticed Sir, the documentation is not yet updated for nx.draw? As I see under ```Draw``` tab , there is both ```draw``` and ```draw_networkx```, saying the latter gives a more full-featured drawing. But afaik, the arguments in ```draw_networkx``` can be used in ```draw``` function as of now. So shouldn't they both be merged? Please correct me if I am wrong @dschult

**Comment by 0ddoes** [comment]
According to Complex Network Theory, I don't think there should be double headed arrows for undirected graphs. If given <->, then there would not be any difference between ```nx.Graph()``` and ```nx.DiGraph()``` once arrows = ```True```. Or in other words, it might remove the difference between undirected and bidirectional graphs. Undirected is supposed to be showing connectedness while directed shows the mutual relations. And while studying actual datasets it could prove a problem. 

My suggestion is that @dtekinoglu could perhaps work on ```nx.draw()``` such that it recognises that it is undirected and gives a default ```None``` value instead. At present it follows 
```python
If `True`, draw arrowheads with FancyArrowPatches (bendable and stylish)
```
Something else I noticed Sir, the documentation is not yet updated for nx.draw? As I see under ```Draw``` tab , there is both ```draw``` and ```draw_networkx```, saying the latter gives a more full-featured drawing. But afaik, the arguments in ```draw_networkx``` can be used in ```draw``` function as of now. So shouldn't they both be merged? Please correct me if I am wrong @dschult

**Comment by rossbar** [comment]
This was fixed by #5514.

There's still the question of whether to raise a warning when a user calls `draw_networkx_edges` with an undirected graph and `arrows=True` as they'll see vastly degraded performance from the fact that edges are drawn with `FancyArrowPatch` instead of `LineCollection` in this case. That's a separate issue though!
